server/cmwell-sparql-agent/src/main/scala/cmwell/tools/data/sparql/SparqlProcessorManager.scala

```diff
@@ -358,6 +358,8 @@ class SparqlProcessorManager (settings: SparqlProcessorManagerSettings) extends
 
     val label = Some(s"ingester-${job.name}")
 
+    val writeToken = if (Some(settings.writeToken).isDefined) Some(settings.writeToken) else None
```

Comment:
Some(*something*) is always a Some and never none (even for null).
This line is practically `writeToken = Some(settings.writeToken)` and those redundant.
See below comment.